backend/molgenis-emx2/src/main/resources/log4j2.xml

```diff
@@ -7,9 +7,9 @@
     </Appenders>
     <Loggers>
         <Logger name="org.pac4j" level="debug" additivity="false">
-            <Appender-ref ref="Console" level="debug" />
+            <Appender-ref ref="Console" level="debug"/>
         </Logger>
-        <Root level="error">
```

Comment:
by accident ? or do you want to change the default ( why and maybe move to separate pr)